src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -1716,6 +1716,21 @@ class device {
         return false;
     }
 
+    bool isAuxConnected(void) {
+        std::string name, errmsg;
+        unsigned short m12v_aux_vol = 0, m3v3_aux_vol = 0;
+
+        pcidev::get_dev(m_idx)->sysfs_get( "rom", "VBNV", errmsg, name );
+        pcidev::get_dev(m_idx)->sysfs_get( "xmc", "xmc_12v_aux_vol",  errmsg, m12v_aux_vol );
+        pcidev::get_dev(m_idx)->sysfs_get( "xmc", "xmc_3v3_aux_vol",  errmsg, m3v3_aux_vol );
+
+        //check aux cable if not u2xx
+        if(strstr( name.c_str(), "_u2" ) && m12v_aux_vol == 0 && m3v3_aux_vol == 0 ) {
```

Comment:
we'll have to match the board name exactly. we'll have u25 soon, which does not have AUX power connectors.
Also, I'm not sure if VBNV name will work well with 2RP in the future. I believe that we can get board name and power level from board info from CMC. So, that probably is a better way of retrieving these information.

Reply:
@maxzhen So I need to check AUX for u250, u200, and u280? are there any other boards? I'll get the name from XMC, but I don't need the power right? I am just checking the voltage for aux cables. Is there something else I should be checking instead?

Reply:
The power level from board info is defined to indicate if aux power is connected. so i think we should look at that one.